src/main/java/org/jboss/forge/arquillian/command/AddContainerDependencyStep.java

```diff
@@ -96,18 +99,33 @@ public Result execute(UIExecutionContext context) throws Exception {
       Map<Object, Object> ctx = context.getUIContext().getAttributeMap();
       Container container = (Container)ctx.get(ContainerSetupWizard.CTX_CONTAINER);
       String version = (String)ctx.get(ContainerSetupWizard.CTX_CONTAINER_VERSION);
+      Project project = getSelectedProject(context);
+      ArquillianFacet arquillian = project.getFacet(ArquillianFacet.class);
+      ArquillianConfig config = arquillian.getConfig();
 
-      containerInstaller.installContainer(
-            getSelectedProject(context),
-            container,
-            version,
-            getVersionedDependenciesMap());
+      final String profileId = container.getProfileId();
+
+      if (container. isSupportedByChameleon(version)) {
+         dependencyManager.addChameleonDependency(project);
+         if (config.isContainerWithDefault("true")) {
+            config.addContainer(profileId);
+         } else {
+            config.addContainerWithAttribute(profileId, "default", "true");
+         }
```

Comment:
If you add multiple containers. then you will get different profiles with different containers, different qaulifier in arqullian.xml, in which only one container has default value.

If user wants to run it with some different container then he has to manually update that respective container with default true.

If you added only single entry then if you have to add configurations for that container then it'll be problematic,